20nov/HW7_task2/tree.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <stdio.h>

```

Comment:
```suggestion
#pragma once

#include <stdio.h>
```
Однако далеко не все подключённые тут хедеры тут на самом деле нужны